ot/optim.py

```diff
@@ -17,9 +17,10 @@
 with warnings.catch_warnings():
     warnings.simplefilter("ignore")
     try:
-        from scipy.optimize import scalar_search_armijo
-    except ImportError:
         from scipy.optimize._linesearch import scalar_search_armijo
+    except ModuleNotFoundError:
+        # scipy<1.8.0
```

Comment:
This comment is added only to make it easier to know when this `try` `except` block can be removed when the `scipy` install_requires lower bound gets raised eventually.